isv/src/prediction.py

```diff
@@ -85,20 +95,22 @@ def predict(annotated_cnv: features.ISVFeatures, cnv_type: annotation.enums.CNVT
     print(f"Loading model from {model_path=}", file=sys.stderr)
     loaded_model = joblib.load(model_path)
 
+    features = loaded_model.feature_names
+    print("Features used in the model:", features)
+
     input_df = prepare_dataframe(annotated_cnv, cnv_type)
 
     dmat_cnvs = xgb.DMatrix(input_df)
     predicted_probability = float(loaded_model.predict(dmat_cnvs)[0])
-    print(f"{predicted_probability=}", file=sys.stderr)
     isv_score = get_isv_score(predicted_probability)
 
-    shap_values = get_shap_values(loaded_model, input_df)
+    shap_values = get_shap_values(loaded_model, input_df, cnv_type)
 
     return Prediction(
         isv_prediction=predicted_probability,
         isv_score=isv_score,
         isv_classification=get_acmg_classification(isv_score),
         isv_shap_values=shap_values,
-        isv_shap_scores=get_shap_scores(shap_values),
+        isv_shap_scores=shap_values,
```

Comment:
je to teda teraz duplikatne? shap_values a shap_scores su zhodne, ma to zmysel ?

Reply:
ano, zatial takto. mozno zatial, aby sa nemusela menit ostatna postupnost - napr v reportingu - ma sa vykreslovat hodnoty zo shap_values (nie dalsie preskalovanie)